app/components/pull-request-page.js

```diff
@@ -630,17 +651,19 @@ export default Ember.Component.extend({
         {{/if}}
       </div>
     {{/unless}}
+    {{#if (eq model.status "approved")}}
```

Comment:
I'd also change this to check if it's rejected. So something like `{{#if (or (eq model.status "approved") (eq model.status "rejected"))}}` (I think rejected is the name of that state, but just double check.

Reply:
Made another {{#if}} statement so that we don't sound so excited when someone's standards are rejected.  I couldn't think of a neutral message that would account for both.